src/test/java/progob/BankTest.java

```diff
@@ -95,4 +95,17 @@ public void testGetClientAccounts()
         assertTrue(bank.getClientAccounts(klient1).contains(konto_Klienta2));
         assertFalse(bank.getClientAccounts(klient1).contains(konto_Klienta3));
     }
+    public void testDeposits() throws WithdrawException
```

Comment:
Test method name should inform what we are testing. Please change test method name for something more meaningful for example: "testGetClientDeposits_When_ClientHasTwoDepositsInBank"

Reply:
I change name of this method